src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+* Fixed an issue where the created activity from ASP.NET Core was replaced
```

Comment:
the changelog entry is hard to follow. Maybe better to link to https://github.com/open-telemetry/opentelemetry-dotnet/issues/4466, and word like
Fix for "IHttpActivityFeature.Activity and Activity.Current don't return the same activity"

Reply:
updated